src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -90,7 +94,7 @@ export function SwapSettingsSlippageInput({
           background.default,
           border.defaultActive,
           'flex h-9 w-24 items-center justify-between rounded-lg border px-2 py-1',
-          isAutoSlippageSetting && 'opacity-50',
+          slippageSetting === SLIPPAGE_SETTINGS.AUTO && 'opacity-50',
```

Comment:
Must be the performance perfectionist in me :)
But everytime you have code that is writtend 2+ times, it cost CPU time. No big deal, but when you see those things, you can just write something like

```
let isSlippageSettingAuto = false;
useEffect(() => {
  isSlippageSettingAuto = slippageSetting === SLIPPAGE_SETTINGS.AUTO;
}, [slippageSetting]); 
```

and just use `isSlippageSettingAuto` evreywher. And make the code more elegant and also more efficient.